commands/verify/verify.go

```diff
@@ -87,6 +88,38 @@ func verifySingleCase(expectedFile, actualFile, query string) error {
 	return nil
 }
 
+type ErrContain struct {
+	errAddr *multierror.Error
+	mutex   sync.Mutex
+}
```

Comment:
1. It would be better to add a comment for `ErrContain`
2. The name `errAddr` is confusing, what does it mean?